src/collections/news/2023-04-12-BBB-Intro.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Introduction of Baby Bliss Bot Project
+date: "2023-04-12"
+excerpt: Advancing Communication for AAC Users through AI Tools
+---
+
+The Baby Bliss Bot project delves into the realm of using AI tools for language development
+and translation to enhance communication for AAC users who rely on a minority language system.
+This article examines the background of Blissymbolics, the potential of AI technology, and
+the objectives of the project.
+
+[Blissymbolics](https://www.blissymbolics.org/), a universal symbol language originally developed
+by Charles K. Bliss for the purpose of international communication, has gained popularity as a
+method for teaching communication to children with physical disabilities who are non-speaking.
+The origin of the use of Bliss as a bridge to communication and literacy can be traced back to the
+pioneer program initiated by Shirley McNaughton at the Ontario Crippled Children's Centre in 1971.
+Blissymbols offer a written semantic language without phonology that serves as a valuable alternative
+for nonvocal individuals, such as those with cerebral palsy, who face obstacles in acquiring phonological
+language skills.
+
+The rapid advancement of AI technology, particularly Large Language Models (LLMs), presents both a
+tremendous opportunity to enhance societal engagement for nonvocal citizens but also comes with many
+risks for a group that is not recognized or included in the training data. The Baby Bliss Bot (BBB)
+project embarks on a community-led investigation to explore the possibilities and challenges of
```

Comment:
Suggest replacement for "embarks on" in "The Baby Bliss Bot (BBB) project embarks on a community-led investigation ..."

Instead:  "The Baby Bliss Bot (BBB) project is a community-led investigation ..."